llvm/lib/Target/RISCV/RISCVISelLowering.cpp

```diff
@@ -8364,6 +8376,27 @@ SDValue RISCVTargetLowering::lowerADJUST_TRAMPOLINE(SDValue Op,
   return Op.getOperand(0);
 }
 
+SDValue RISCVTargetLowering::lowerPARTIAL_REDUCE_MLA(SDValue Op,
```

Comment:
It looks like there's tablegen nodes defined for partial_reduce_{u,s}mla, could we mark the node as legal instead of custom and patterns in tablegen instead?

Reply:
I briefly looked at doing this via tablegen, but decided to share the code with the reduce pattern matching.  Once I get hat migrated over to the partial_reduce_mla infrastructure, I may revisit the tablegen question.